src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -405,6 +405,34 @@ int NonClientIslandWindow::_GetResizeHandleHeight() const noexcept
     return HTCAPTION;
 }
 
+// Method Description:
+// - Gets the difference between window and client area size.
+// Arguments:
+// - dpi: dpi of a monitor on which the window is placed
+// Return Value
+// - The size difference
+SIZE NonClientIslandWindow::GetClient2WindowSizeDelta(UINT dpi) const noexcept

```

Comment:
Honestly I still don't _love_ the name of this, but I can't really think of a clear better name. `GetNonClientBorderDimensions`? `GetNonClientFrameSize`?

Maybe @DHowett-MSFT or @miniksa might have thoughts